Submissions/michymichelle.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Michelle Zheng",
+  "img": "[YOUR_IMG_URL]",
```

Comment:
Please fix your placeholders. You can read the `README` if you don't know what I mean. The guide explains what to do.